src/Mailer/SettingsMailer.php

```diff
@@ -169,137 +172,151 @@ public function getSettingsData(string $formId): array
 					[
 						'component' => 'checkbox',
 						'checkboxLabel' => \__('Use Mailer', 'eightshift-forms'),
-						'checkboxIsChecked' => $this->isCheckboxSettingsChecked(self::SETTINGS_MAILER_USE_KEY, self::SETTINGS_MAILER_USE_KEY, $formId),
+						'checkboxIsChecked' => $isUsed,
 						'checkboxValue' => self::SETTINGS_MAILER_USE_KEY,
 						'checkboxSingleSubmit' => true,
+						'checkboxAsToggle' => true,
+						'checkboxAsToggleSize' => 'medium',
 					]
 				]
 			],
-		];
-
-		if ($isUsed) {
-			$formNames = Helper::getFormNames($formId);
-
-			$output = \array_merge(
-				$output,
-				[
-					[
-						'component' => 'divider',
-					],
+			$isUsed ? [
+				'component' => 'tabs',
+				'tabsContent' => [
 					[
-						'component' => 'intro',
-						'introTitle' => \__('E-mail settings', 'eightshift-forms'),
-						'introTitleSize' => 'medium',
+						'component' => 'tab',
+						'tabLabel' => \__('Settings', 'eightshift-forms'),
+						'tabContent' => [
+							[
+								'component' => 'input',
+								'inputName' => $this->getSettingsName(self::SETTINGS_MAILER_SENDER_NAME_KEY),
+								'inputId' => $this->getSettingsName(self::SETTINGS_MAILER_SENDER_NAME_KEY),
+								'inputFieldLabel' => \__('Sender name', 'eightshift-forms'),
+								'inputFieldHelp' => \__('Most e-mail clients show this instead of the e-mail address in the list of e-mails.', 'eightshift-forms'),
```

Comment:
```suggestion
								'inputFieldHelp' => \__('Most e-mail clients show the sender name instead of the e-mail address in the list of e-mails.', 'eightshift-forms'),
```